src/index.js

```diff
@@ -3,18 +3,22 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
-import {BrowserRouter} from 'react-router-dom';
+import { BrowserRouter } from "react-router-dom";
+import { FilterProvider } from "./context/FilterContext";
+import { ProductProvider } from "./context/ProductContext";
 
```

Comment:
I would suggest you to use barrel exports as it will be easier for later developments and easier to debug